<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userDetails</class>
 <widget class="QWidget" name="userDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>userDetails</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#userDetails {
background-color: rgb(149, 99, 74);
}</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>302</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="verticalSpacing">
     <number>24</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font-size:20px;
color:white;</string>
      </property>
      <property name="text">
       <string>Employee Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="emp_name">
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border:1px solid white ;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font-size:20px;
color:white;</string>
      </property>
      <property name="text">
       <string>Employee Id</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="emp_id">
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border:1px solid white ;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font-size:20px;
color:white;</string>
      </property>
      <property name="text">
       <string>Employee Type</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="emp_type">
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border:1px solid white ;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font-size:20px;
color:white;</string>
      </property>
      <property name="text">
       <string>Date Of Birth</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="emp_dob">
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border:1px solid white ;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font-size:20px;
color:white;</string>
      </property>
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="emp_number">
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border:1px solid white ;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>620</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QPushButton" name="details_edit_btn">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
color:black;
background-color: white;</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_btn">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>9</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
color: rgb(255, 255, 255);
background-color: rgb(135, 0, 0);</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="show_item_widget" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>720</y>
     <width>471</width>
     <height>251</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>399</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>24</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font-size:20px;
color:white;</string>
       </property>
       <property name="text">
        <string>Which one Do you want to Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>select</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Employee type</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Date Of Birth</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ph_number</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>251</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="details_ok_btn">
       <property name="styleSheet">
        <string notr="true">font-size:20px;
color:black;
background-color: white;</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="details_cancel_btn">
       <property name="styleSheet">
        <string notr="true">font-size:20px;
color:black;
background-color: white;</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>391</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>88</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">font-size:20px;
color:white;</string>
       </property>
       <property name="text">
        <string>Enter the New Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="showwidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="list_widget" native="true">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>680</y>
     <width>171</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Developer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tester</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Manager</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Intern</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>851</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:30px;
color:white;</string>
    </property>
    <property name="text">
     <string>Welcome username!..</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HomeWindow.qrc"/>
 </resources>
 <connections/>
</ui>
